substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/pecoff/cv/CVLineRecordBuilder.java

```diff
@@ -85,26 +85,26 @@ private void processRange(Range range) {
 
         FileEntry file = range.getFileEntry();
         if (file == null) {
-            debug("processRange: range has no file: %s\n", range);
+            debug("  processRange: range has no file: %s", range);
             return;
         }
 
-        if (range.getLine() == -1) {
-            debug("processRange: ignoring: bad line number\n");
+        if (range.getLine() < 0) {
+            debug("  processRange: ignoring: bad line number: %d", range.getLine());
             return;
         }
 
         int fileId = cvDebugInfo.getCVSymbolSection().getFileTableRecord().addFile(file);
         if (lineRecord.isEmpty() || lineRecord.getCurrentFileId() != fileId) {
-            debug("processRange: addNewFile: %s\n", file);
+            debug("  processRange: addNewFile: %s", file);
             lineRecord.addNewFile(fileId);
         }
 
         /* Add line record. */
         /* An optimization would be to merge adjacent line records. */
         int lineLoAddr = range.getLo() - primaryEntry.getPrimary().getLo();
         int line = Math.max(range.getLine(), 1);
-        debug("processRange:   addNewLine: 0x%05x %s\n", lineLoAddr, line);
+        debug("  processRange:   addNewLine: 0x%05x-0x%05x %s", lineLoAddr, range.getLo() - primaryEntry.getPrimary().getHi(), line);
```

Comment:
I'm not quite sure what `range.getLo() - primaryEntry.getPrimary().getHi()` is supposed to be. Did you perhaps mean `range.getHi() - primaryEntry.getPrimary().getLo()` instead?

Reply:
Yes, I did - good catch.  Fixed.